build/.goreleaser.yml

```diff
@@ -41,6 +41,7 @@ builds:
       - linux_amd64
       - linux_arm64
       - linux_ppc64le
+      - linux_s390x
```

Comment:
We cannot support the release of the tool for `linux_s390x` if our envtest binaries does not run in this ENV.
See that we build them here: https://github.com/kubernetes-sigs/kubebuilder/tree/tools-releases and we would need to setup a GCP Build target to begin to support this one.

Also, will ALL images build and ship work within ? We either need to check it?